SeedV/SeedUnityVRKit
# Upgrade MediaPipeUnityPlugin to v0.9.0.

Suggestions on how to review this change:
1. Review changes in `PoseTracker`
2. Verify the move of plugin folder.
3. Verify integrity of the change, e.g. checkout and load the project in unity and load the scene. I have only been able to support M1 Mac though, so Intel Mac should build the plugin from source in upstream project and put the created `libmediapipe_c.dylib` in `Packages/com.github.homuler.mediapipe/Runtime/Plugins`.

Sorry for the very large change. There are pretty of incompatible changes due to development of [github.com/homuler/MediaPipeUnityPlugin](https://github.com/homuler/MediaPipeUnityPlugin). The main ones are:

- Check out [MediaPipeUnityPlugin@v0.9.0](https://github.com/homuler/MediaPipeUnityPlugin/tree/v0.9.0) and recompile the plugin on M1 (Apple Silicon).
- Pull in new C# scripts from its sample project.
- Replace `PoseTrackingGraph` with `HolisticTrackingGraph`. Update `PoseTracker` script to use the new graph.
- Update `PoseTracker` due to incompatible API changes.
- Update some upstream code to allow classes like `PoseTracker` to create SidePacket and send it to the graph.
- Removed unrelated models in `StreamingAssets`.
- Move the M1 plugin under `MacOS/arm64`.
- Move the plugin package from `Assets/Plugins` to top-level `Packages/`.

It also upgraded Editor version to 2021.3.0f1.

Changes are tested in dev branch to make sure the scene is loaded and functionality is not broken.
